lib/hrr_rb_ssh/open_ssl_compat.rb

```diff
@@ -0,0 +1,32 @@
+module HrrRbSsh
+  module OpenSSLCompat
```

Comment:
hrr_rb_ssh has a directory/module hrr_rb_ssh/compat for compatibility.
Would you please put this module in the directory ?

Reply:
Thanks; Swapped things around and squashed the commit :+1: